lib/samlp.js

```diff
@@ -145,6 +148,7 @@ function buildSamlResponse(options) {
 
 function getSamlResponse(options, user, callback) {
   options.profileMapper = options.profileMapper || PassportProfileMapper;
+  options.prefix = options.prefix && (typeof options.prefix === 'string') ? options.prefix : '' ;
```

Comment:
`(typeof options.prefix === 'string')` instead of `options.prefix && (typeof options.prefix === 'string')` ?
Anyway, is it not enough the basic sanitize: `options.prefix = options.prefix || ''` ?